staff/mykhaylo-gusak/cv/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Mykhaylo Gusak</title>
+</head>
+
+<body>
+
+    <header>
+        <h1>Currículum</h1>
+        <section>
+            <h2>Datos personales</h2>
+            <p><b>Nombre:</b>Mykhaylo Gusak</p>
+            <p><b>Edad:</b>27</p>
+            <p><b>Ciudad:</b>Barcelona</p>
+            <h2>Foto</h2>
+            <img src="https://ca.slack-edge.com/T0SJKHBFZ-UFGEK7P7H-fbcf35506193-150" alt="Mykhaylo Gusak">
+        </section>
+    </header>
+
+    <section>
+        <h2>Sobre mi:</h2>
+        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Repudiandae vitae in ut aliquam tempora perspiciatis quia vero consectetur nostrum distinctio officiis consequuntur architecto veniam, minus iste facilis, ex at saepe temporibus eligendi aperiam mollitia obcaecati?</p>
+    </section>
+
+    <section>
+        <header>
+            <h2>Estudios</h2>
+        </header>
+
+        
+
+        <article>
+            <h3>Instituto IES Torre Vicens</h3>
+            <p>Años:</p>
+        </article>
+
+        <article>
+                <h3>Universidad de Lleida</h3>
+                <p>Años:</p>
+        </article>
+
+
+    </section>
+
+    <section>
+
+        <header>
+            <h3>Experiencia</h3>
+        </header>        
+        
+        <article>
+            <h3>Geodisa S.L.</h3>
+            <p>Años:</p>
+        </article>
+
+        <article>
+            <h3>JAFY Architecture </h3>
+            <p>Años:</p>
+        </article>
+
+        <article>
+            <h3>Labox Design S.L.P.</h3>
+            <p>Años:</p>
+        </article>
+
+        <article>
+            <h3>Bootcamp: Skylab coders</h3>
+            <p>Año:2019</p>
+        </article>
+
+    </section>
+
+    <section>
+
+        <header>
+            <h2>Idiomas:</h2>
+        </header>
+
+        <!-- 
```

Comment:
@MichaelXXI remove this comment if it is not necessary

Reply:
@robert-z Comment already removed.